docs/en/compute/AWS-EMR.md

```diff
@@ -44,8 +44,8 @@ To begin with, [download an Alluxio release](https://www.alluxio.io/download) an
 $ aws emr create-default-roles
 ```
 2. The Alluxio bootstrap script is hosted in a publicly readable
-[S3 bucket](https://alluxio-public.s3.amazonaws.com/emr/2.0.1/alluxio-emr.sh).
-This bucket can also be accessed using it's S3 URI: `s3://alluxio-public/emr/2.0.1/alluxio-emr.sh`
+[S3 bucket](https://alluxio-public.s3.amazonaws.com/emr/{{site.ALLUXIO_RELEASED_VERSION}}/alluxio-emr.sh).
```

Comment:
curious is this will always be true...

Reply:
@Xenorith @tieujason330 Please comment on this? It's an AWS artifact.

Reply:
lgtm